client/translations/en.ts

```diff
@@ -818,7 +818,7 @@ Control code: %1</translation>
     </message>
     <message>
         <source>Sign certificate is not selected</source>
-        <translation>Sign certificate is not selected</translation>
+        <translation>Signature certificate is not selected</translation>
```

Comment:
I think it should be "Signing certificate" instead
(sorry for the bad advice before)